user-guide.adoc

```diff
@@ -0,0 +1,44 @@
+== About Hyperscience
+
+The vast majority of information that organizations need to power processes, service customers,
+and drive business outcomes lives trapped in unstructured formats, including PDFs, documents,
+and images. With the Hyperscience Intelligent Document Processing Platform, clients set their
+target level of accuracy (typically over 99% based on internal SLAs or other compliance requirements),
+and our system automates against that using our proprietary ML. Hyperscience reads outside the box
+to handle messy handwritten forms, low-resolution images, and other document distortions with ease,
+delivering best-in-class intelligent data extraction.
+
+Our proprietary Machine Learning approach improves throughput without sacrificing accuracy. We’re
+exceptionally good at knowing when to bring in humans to review and resolve edge cases, which helps
+finetune the underlying models. Hyperscience continues to learn, lowering error rates and increasing
+automation over time.
+
+The interface of the Hyperscience Platform allows humans to teach Hyperscience and process submissions
+with ease. Our Intelligent Document Processing (IDP) platform was designed to be used by business users
+with no technical experience required. Hyperscience’s fast time-to-value is influenced by the short time
+it takes to deploy the system and integrate it within an organization’s environment.
+
+For more information, visit www.hyperscience.com.
+
+== About the Hyperscience Connector
+
+The Hyperscience connector enables easy integration between MuleSoft and Hyperscience for the purposes
+of content processing.
+
+== Associated Use Cases
+
+- Create submissions
+- Retrieve submission
+- List submissions
+
+== Contact Us
+
+Tim Wikander <tim.wikander@hyperscience.com>, Director, Corporate Development
```

Comment:
@kand @rui-lomba do we want to have Tim here? What happens if he leaves the company? Shouldn't we have a more general contact email/persona?

Reply:
Yea we should definitely have a more general contact. I'll message Tim and see if he's got an idea of an email we can use.

Reply:
https://hyperscience.slack.com/archives/C02L4U15Y14/p1669933189328109

Reply:
From all the examples I saw online e.g. [Amazon Connector](https://www.mulesoft.com/exchange/com.mulesoft.connectors/mule-amazon-s3-connector/minor/6.0/), they do not share any contact so I removed this section

Reply:
This section was moved to Home page instead of the user guide as per discussed [here](https://hyperscience.slack.com/archives/C04BQJ4V8UW/p1669983481170599).

This section was moved to the Home.md page on the connector and the contact will be
[integrations@hyperscience.com](mailto:integrations@hyperscience.com)